src/index.js

```diff
@@ -3,5 +3,14 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
+import {
+    BrowserRouter as Router,
```

Comment:
Good job here.  You don't necessarily need Route or Link inside the index.js file,  but its not hurting anything either.